### DISCUSSION TITLE
high network traffic of SMplayer while playing a locally stored video

### DISCUSSION THREAD

**Discussion by gilunixuser** [smplayer-dev/smplayer#815] [General]
I am noticing a high network traffic of SMplayer while playing a locally stored video.
That should not be the case. I am not happy if my data are sent anywhere.
Can this network connection and data transfer of smplayer be turned off anywhere?

**Comment by smplayer-dev** [comment]
SMPlayer just connects on startup to the website to check if there's any new version available. That should only be a few bytes.

**Comment by gilunixuser** [comment]
Even if the turning on and turning off of smplayer corresponds with the network traffic but according to your statement it must be still some other reason than. I believe in your words. Thank you.
